03 Writing Algorithms/03 Securities/01 Key Concepts/03 Holdings.html

```diff
@@ -1,17 +1,24 @@
 <p>The <code class="csharp">holdings</code><code class="python">Holdings</code> property of a <code>Security</code> object contains information about the <a href='/docs/v2/writing-algorithms/portfolio/key-concepts'>investment state and history</a> of the security.</p>
 
 <div class="section-example-container">
-    <pre class="csharp">var holdings = Securities[_symbol].Holdings;
-var quantity = holdings.Quantity;</pre>
-    <pre class="python">holdings = self.securities[self._symbol].holdings
-quantity = holdings.quantity</pre>
+    <pre class="csharp"> // Get the quantity of currently holded securities
```

Comment:
- Please use full stops, check for typos. 
 - Don't use the word you're defining in the comment. Held-holdings.

E.g. // Use holdings object for information on asset positions such as quantity, and average price.